tests/integration_tests/functional/test_rng.py

```diff
@@ -208,16 +216,14 @@ def _rate_limiter_id(rate_limiter):
     indirect=True,
     ids=_rate_limiter_id,
 )
-def test_rng_bw_rate_limiter(uvm_with_rng):
+@pytest.mark.parametrize("uvm_fun", [uvm_with_rng_booted], indirect=True)
+def test_rng_bw_rate_limiter(uvm_any):
```

Comment:
This doesn't test work on restored uVMs? Or we just don't need to test it on restored ones?

Reply:
got it, leave it as it is here 👍